DEVELOP.md

```diff
@@ -73,6 +73,7 @@ gulp                    # Compile the JS and start the automatic
 To regenerate the `bundled.js` file without using the live reloader, run:
 
 ```
+gulp peg
```

Comment:
Maybe stick this between line 68 and 68 with a note that it only needs to be run when changing `querylanguage.pegjs`?